shell/browser/native_window_mac.mm

```diff
@@ -1345,15 +1345,37 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
   original_frame_ = [window_ frame];
 }
 
-void NativeWindowMac::SetVibrancy(const std::string& type) {
+void NativeWindowMac::SetVibrancy(const std::string& type,
+                                  bool animate,
+                                  int duration) {
   NSVisualEffectView* vibrantView = [window_ vibrantView];
 
   if (type.empty()) {
     if (vibrantView == nil)
       return;
 
-    [vibrantView removeFromSuperview];
-    [window_ setVibrantView:nil];
+    vibrancy_type_ = type;
+
+    if (animate) {
+      __weak auto weak_delegate = window_delegate_.get();
```

Comment:
Please write the type out instead of `auto` here. Generally our style is to only use `auto` when the full type is already written somewhere else on the same line.

Reply:
[fixed](https://github.com/electron/electron/pull/35987/commits/310ca9f9516497f81849c4ae56afbb19f8805213)